lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/left_side_navigation.rb

```diff
@@ -0,0 +1,38 @@
+require 'httparty'
```

Comment:
In general - I don't think adding page object for link without new test is a good idea.
This objects can be added in new PR with test, since now some of xpaths may be incorrect